src/track/trackmetadatataglib.cpp

```diff
@@ -1256,6 +1340,11 @@ void readTrackMetadataFromMP4Tag(TrackMetadata* pTrackMetadata, const TagLib::MP
 #endif
     }
 
+	QString rating;
+	if (readMP4Atom(tag, "rate", &rating)) {
+	    parseRating(pTrackMetadata, rating, FileType::MP4);
```

Comment:
As already mentioned, the FileType is always known at compile time and does not need to passed as an additional argument.